mvcc/backend/backend.go

```diff
@@ -163,6 +165,18 @@ func (b *backend) ForceCommit() {
 }
 
 func (b *backend) Snapshot() Snapshot {
+	// tracks duration of snapshot()
+	stopc, donec := make(chan struct{}), make(chan struct{})
+	go func() {
+		defer close(donec)
+		now := time.Now()
+		select {
+		case <-time.After(snapshotWarningTimeout):
```

Comment:
yes. probably also log when it starts to differentiate concurrent ones.

snapshot is taking more than ... [started at ]